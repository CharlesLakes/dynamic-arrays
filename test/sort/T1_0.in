10
32 68 82 64 80 23 44 95 34 28