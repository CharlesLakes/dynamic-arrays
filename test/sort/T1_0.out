23 28 32 34 44 64 68 80 82 95